### PR TITLE
Change interfaces from MAPMLElement to HTMLElement

### LINKED ISSUE #73: Interfaces for elements should be (or extend) HTMLElement
> > Interfaces for elements should be HTMLElement, or HTMLExtentElement : HTMLElement etc.
> 
> Does this need to be specified in the MapML specification/proposal? I think yes, but not sure.

Yes.

_Originally posted by @zcorpan in https://github.com/Maps4HTML/MapML/issues/70#issuecomment-653692397_

### PR DESCRIPTION
**Pull request by Malvoz**
Fix https://github.com/Maps4HTML/MapML/issues/73.

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -1428,7 +1428,7 @@ <h5>The <code>&lt;<dfn id="the-mapml-element">mapml</dfn>&gt;</code> element</h5
           <dd>
             <pre class="idl">
             [Exposed=Window]
-            interface MAPMLMapMLElement : MAPMLElement {
+            interface HTMLHtmlElement : HTMLElement {
```
Comment (prushforth) [inline spec/index.html]: I think at the moment this should be "interface HTMLMapmlElement : HTMLElement".  @zcorpan and I did not close the loop on this conversation, but the reasoning is that the <mapml> root document element would still be necessary for a file that is read from a file system and assigned to the browser application by its "filename.ext" file system file type extension mapping, but for which the browser has no previous hint to go by, such as a <!DOCTYPE ...> declaration (In such a case the browser would be expected, as it does with, for example, "myfile.png", synthesize an HTML document enclosing the markup that would render the map layer with default starting conditions, if possible).

**Review by prushforth — CHANGES_REQUESTED** [review]
Thanks for doing this!  It will help the professional reviewers avoid confusion, to some extent :-).

Please see the comment I left; apart from that LGTM.

File: spec/index.html (hunk 1 above)
Comment (Malvoz) [reply spec/index.html]: Okay, changed interface to `HTMLMapmlElement` in https://github.com/Maps4HTML/MapML/pull/142/commits/0de8500ac997ea223c56df94b6ba9b1998cdf107.

Comment (Malvoz) [reply spec/index.html]: Another thing though, we're currently describing the `map` element as a media element:

> The [`map`](https://maps4html.org/MapML/spec/#the-map-element) element, in conjunction with child [`layer`](https://maps4html.org/MapML/spec/#the-layer-element) and optionally [`area`](https://maps4html.org/MapML/spec/#the-area-element) elements, defines a map [media element](https://html.spec.whatwg.org/multipage/media.html#media-element).

That means the element interface for `<map>` should be "HTMLMapElement : [HTMLMediaElement](https://html.spec.whatwg.org/multipage/media.html#htmlmediaelement)".

But, I am responsible for linking to the "[media element](https://html.spec.whatwg.org/multipage/media.html#media-element)" WHATWG definition, which I first should ask, did you intend to define the map element as a "media element" per that definition (with the presumption that the definition would be extended to include `map`)?

**Comment by prushforth** [comment]
@Malvoz no I think the HTMLMediaElement interface would be incorrect for the map element. The use of the term 'media' here refers to MIME media types, at most, although it's not clear thattjeterm should be use in this context. I will review and reword.

**Comment by Malvoz** [comment]
@prushforth Okay, sorry for making such an assumption.

Feel free to merge/approve this if you think it's ready, and we can deal with re-wording/remove links to the "media element" definition separately.

**Pull request merged**